### ISSUE TITLE
apl.interactive.plot is not working anywhere

### ISSUE THREAD

**Issue by denizsincar29** [hassaku/audio-plot-lib#2]
Hello. Tryed to use interactive plot in python and in colab, both don't work.
AttributeError                            Traceback (most recent call last)
<ipython-input-4-8b3c1beb48ce> in <cell line: 1>()
----> 1 apl.interactive.plot([0, 1, 2, 3, 2, 1, 0])

 
2 frames
 
/usr/local/lib/python3.10/dist-packages/bokeh/core/has_props.py in _raise_attribute_error_with_matches(self, name, properties)
    366             matches, text = sorted(properties), "possible"
    367 
--> 368         raise AttributeError(f"unexpected attribute {name!r} to {self.__class__.__name__}, {text} attributes are {nice_join(matches)}")
    369 
    370     def __str__(self) -> str:

AttributeError: unexpected attribute 'plot_width' to figure, similar attributes are outer_width, width or min_width

**Comment by hassaku** [comment]
Thank you so much for reporting the issue and sorry for the inconvenience.
I guess it is probably due to an upgrade of a library that depends on it.
I will attempt to fix it this weekend and will mention it to you.

**Comment by denizsincar29** [comment]
oh thank you! I am very very excited to try it out, but can I pip install the old library as a workarround?

**Comment by denizsincar29** [comment]
Actually i also want an image explorer that turns the image into sinewaves. e.g. if there is a circle image, 2 sinewaves are plotting the circle shape (one goes up, other goes down. and than both go to the same point, representing circle).
i'd like to open a png or svg image and view it's soundplot.

**Comment by denizsincar29** [comment]
@hassaku do you have any simple demo interactive html of a graph while the library's not working? wanna check it out

**Comment by hassaku** [comment]
Some simple demo interactive html can be found at the followings:
- https://a11y-ds-intro.hassaku-labs.com/chapter-3/audio-plot-lib.html
- https://hassaku.github.io/DS-and-ML-with-screen-reader/

I would be happy to help.

**Comment by denizsincar29** [comment]
@hassaku The html code is very big and difficult to understand. I thaught it would all be 1 small js library that makes all work with sliders and graphs and i thaught it would be that easy as this:
```html
<script src="path/to/audioplotlib.js"></script>
<script>
apl=new AudioPlotLib("id of the <div>", x=[-5,-4,-3,...], y=[...]);
</script>
```
and it would automatically add the unmute button, make a slider and draw the graph in canvas for not blind people!
well, i'm just sharing idea.

**Comment by hassaku** [comment]
@denizsincar29 
Sorry it took so long to fix, I hope #4 has resolved the error reported in this Issue. 
Please try it when you have time.

And I'm sorry that this library is not very simple, as it is very dependent on the Bokeh functionality.

**Comment by denizsincar29** [comment]
well, it's working fine
